lib/rex/post/meterpreter/packet_dispatcher.rb

```diff
@@ -579,8 +579,7 @@ def decrypt_inbound_packet(packet)
   def dispatch_inbound_packet(packet)
     handled = false
 
-    # Uncomment this line if you want to see inbound packets in the console
-    # STDERR.puts("\n\e[1;32mRECV\e[0m: #{packet.inspect}\n")
+    log_packet(packet, :inbound)
```

Comment:
I prefer your name, but for consistency:

```suggestion
    log_packet(packet, :recv)
```